### PR TITLE
WIP Replace apostrophe with quotes in frontend

### PR DESCRIPTION
**Pull request by fzzzy**

### CODE REVIEW

File: frontend/tasks/util.js [hunk 1]
```diff
@@ -46,13 +46,13 @@ function experimentL10nId(experiment, pieces) {
...
-  if (typeof pieces === 'string') {
+  if (typeof pieces === "string") {
     pieces = [pieces];
   }
   const piecesOut = [];
-  for (let i=0; i<pieces.length; i++) {
+  for (let i = 0; i < pieces.length; i++) {
     piecesOut.push(pieces[i]);
-    const suffix = lookup(experiment, `${pieces.slice(0, i+1).join('.')}_l10nsuffix`);
+    const suffix = lookup(experiment, `${pieces.slice(0, i + 1).join(".")}_l10nsuffix`);
```
Comment (lmorchard) [inline frontend/tasks/util.js:55]: Oh hey, I found a non-quote change! (It's fine though)

Comment (fzzzy) [reply frontend/tasks/util.js:55]: Wow, sharp eye there.

**Review by lmorchard — APPROVED** [review]
Me from IRC:
```
<@lorchard> I'm sure I blacked out somewhere in the middle of looking through the quotes lint PR, but 
<@lorchard> it's looking good to me
<@lorchard> Saw the conflict resolutions and those are good too
<@lorchard> Hardest part was going to the bottom and clicking on all the "load diff" boxes in github because even github doesn't want to go through the effort
```

**Comment by fzzzy** [comment]
Well let's get it over with, like ripping a bandaid off real fast

**Pull request merged**